### PR TITLE
Added divide function to arithmetic_operations

### PR DESCRIPTION
**Pull request by Ajisco**

### CODE REVIEW

File: ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py
Comment (juliagsy) [inline ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py:36]: over-indented

File: ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py
Comment (juliagsy) [inline ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py:39]: `ret` instead of `div` as you're using the `ret` variable in the `ivy.where` argument

File: ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py
Comment (juliagsy) [inline ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py:41]: `ret` instead of `div` as you're using the `ret` variable in the `ivy.where` argument

File: ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py
Comment (juliagsy) [inline ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py:42]: `ret` instead of `div` as you're using the `ret` variable in the `ivy.where` argument

File: ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py
Comment (juliagsy) [inline ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py:46]: please move this above so that it is below the `add` function

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey! Great work there! There are a few things to amend before it is good to go! Please request my review when you're done, thanks for contributing!

**Comment by Ajisco** [comment]
Hello thanks for the feedback. I made the corrections. Kindly review.

**Comment by Ajisco** [comment]
> Hey! Great work there! There are a few things to amend before it is good to go! Please request my review when you're done, thanks for contributing!

Thanks for the feedback.
I already made the corrections.
Kindly review.

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hello! Looks great so far! Please also create frontend tests for this function in the `ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_arithmetic_operations.py` file, thank you!

**Comment by Ajisco** [comment]
> Hello! Looks great so far! Please also create frontend tests for this function in the `ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_arithmetic_operations.py` file, thank you!


I already created Front-end test for the function as asked.
Kindly review:

https://github.com/unifyai/ivy/pull/2629

Thanks.

**Comment by juliagsy** [comment]
oh! I see, thanks for linking the PR! The assignee for your other PR will review as soon as possible! Meanwhile, I'll merge this PR of yours first, thanks for contributing!

**Pull request merged**
